engine/apps/api/views/schedule.py

```diff
@@ -219,6 +227,7 @@ def get_object_from_organization(self):
         organization = self.request.auth.organization
         queryset = organization.oncall_schedules.filter(
             public_primary_key=pk,
+            *self.available_teams_lookup_args,
```

Comment:
Same case as with custom buttons. This method is used to show selected object in escalation step if this object belongs to other team (case when escalation chain was moved to another team). I don't think that we should filter by available teams here 🤔

Reply:
we will display this kind of schedules as "🔒private schedule"